modules/related-posts/jetpack-related-posts.php

```diff
@@ -1639,7 +1658,9 @@ class_exists( 'Jetpack_AMP_Support' )
 	 * @return null
 	 */
 	protected function _action_frontend_init_page() {
-		$this->_enqueue_assets( true, true );
+
+		$enqueue_script = ! ( class_exists( 'Jetpack_AMP_Support' ) || Jetpack_AMP_Support::is_amp_request() );
```

Comment:
Should be an AND operator, no?

```suggestion
		$enqueue_script = ! ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() );
```

Or alternatively:

```suggestion
		$enqueue_script = ! class_exists( 'Jetpack_AMP_Support' ) || ! Jetpack_AMP_Support::is_amp_request();
```